wazuh/wazuh-kibana-app
# Fix flyout date

Hi team this PR solves:
- Flyout date filter also changes main date filter

### How to test
To test this PR you should go to MITRE section an then click on a technique. When the flyout is displayed you have to change the time filter and it should change but the main time filter shall remain as before.


https://user-images.githubusercontent.com/46006326/113568213-67cc0880-9610-11eb-9f03-2c923039183b.mov

Closes #3040
